administrator-manual/en/nsent/_static/nethesis.css

```
@@ -1,43 +1,51 @@
 @import 'https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300,700';
 body {
-    color: #333;
-    background-color: #efefef;
-    font-size: 15px;
+    color:  	#5B5B5B;
+    background-color: white;
+    font-size: 17px;
     line-height: 24px;
 }
 .container {
-    background-color: #fff;
+    background-color:  	#B0C4DE;
```

Do not insert TAB characters. For instance here (and also below)

```suggestion
    background-color: #B0C4DE;
```